MinderaPeopleTests/Dependencies/BiometricAuthenticator/LAContextMock.swift

```diff
@@ -0,0 +1,15 @@
+import LocalAuthentication
+import Foundation
+
+class LAContextMock: LAContext {
+    var canEvaluatePolicyResponse = true
+    var evaluatePolicyResponse = true
+    
+    override func canEvaluatePolicy(_ policy: LAPolicy, error: NSErrorPointer) -> Bool {
+        return canEvaluatePolicyResponse
+    }
+    
+    override func evaluatePolicy(_ policy: LAPolicy, localizedReason: String) async throws -> Bool {
+        return evaluatePolicyResponse
+    }
```

Comment:
some important test are missing. we want to make sure that the LAcontext is being used and providing us the real authentication right? 
so we should create a couple of variables to test and make sure this is being called

```suggestion
    
    private(set) var canEvaluatePolicyCalledCount = 0
    override func canEvaluatePolicy(_ policy: LAPolicy, error: NSErrorPointer) -> Bool {
        canEvaluatePolicyCalledCount += 1
        return canEvaluatePolicyResponse
    }
    
    private(set) var evaluatePolicyCalledCount = 0
    override func evaluatePolicy(_ policy: LAPolicy, localizedReason: String) async throws -> Bool {
        evaluatePolicyCalledCount += 1
        return evaluatePolicyResponse
    }
```

this way you can go to BiometricAuthenticatorTests and check if we have called this method...

Reply:
we can even get beyond and store what kind of policy are we asking and make a test to check if is the right one... 
```swift
private(set) var lastCanEvalutePolicyLAPolicy: LAPolicy!
```